app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,121 @@
+KONG_PG_RO_IAM_AUTH=on
+```
+
+### Configuration file
+
+The `kong.conf` file contains the property `pg_iam_auth` and `pg_ro_iam_auth`.
+Just like the environmen variable, you can set them to `on` accordingly, if you want to enable the IAM Authentication on both read and write connection, or just read-only connection to the RDS Postgres database.
+
+```text
+# Set pg_iam_auth if you want to enable the feature on both read and write
+pg_iam_auth=on
+# Set pg_ro_iam_auth if you only want to enable on the read only connection
+pg_ro_iam_auth=on
+```
+
+{:.note}
+> **Note:** If you're enabling the feature in the configuration file, don't forget to specify the configuration file with the feature property on when you're running migrations command! For example `kong migrations bootstrap -c /path/to/kong.conf`
+
+## Limitations
+
+The AWS IAM Authentication also has some limitations, some are from the AWS service and some are within the Kong runtime. Be sure to go through them one by one before you start to use this feature in your production environment.
+
+- According to the AWS documentation, the AWS IAM Authentication to the RDS will work "when your application requires fewer than 200 new IAM database authentication connections per second". Establishing database connections more than that number in every second may result in connection throttling, controlled by the IAM service. Note that this does not affect the total number of keep-alive connections from Kong to the database, it only affects the connection establishment. Especially if you're using a traditional cluster(in which every Kong node needs to connect to the database) deployment with many Kong nodes, make sure that your number of connection establish action in your whole cluster is lower than the maximum value of 200.
+- Enabling the AWS IAM Authentication feature requires mandatory SSL connection to the database, which needs you to configure both your RDS cluster properly, and provide correct ssl related configs on Kong's side. Enabling SSL will also introduce some performance overhead(but not much) if you do not use SSL connection to your database in the old time.
+- Since the Postgres RDS does not support mTLS, enabling mTLS between the Kong and the Postgres RDS database is not allowed when using this feature.
+- After enabling this feature on the read/write mode or on the read-only mode, the `pg_password` or `pg_ro_password` configuration will be ignored accordingly when establishing connections.
+- As said in the Prerequisition section, you **cannot** change the value of the environment variables that used to provide the AWS Credential after Kong instance boot up. The change will be ignored and **will not take effect** inside Kong instance.
+- As said in the Prerequisition section, a database user with `rds_iam` role granted cannot authenticate in the normal username/password way, you must use the IAM Database authentication instead.
```

Comment:
We could mention in this list one more time that pg_ssl `tlsv1_3` is not supported

Reply:
Done